ClassicLibrary.c

```diff
@@ -4,6 +4,27 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // ClassicLibrary.c
-// Program on branch main.
+// Program on branch A4.
 // Program that contains the definiton of functions defined in Header.h.
-// ------------------------------------------------------------------------------------------------
\ No newline at end of file
+// ------------------------------------------------------------------------------------------------
+#pragma warning(disable:4996)
+#include <string.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <limits.h>
+#include <ctype.h>
+#include "Header.h"
+
+bool IsPalindrome (char* input) {
+   if (input[0] == '\0') return false;
+   for (int end = strlen (input) - 1, start = 0; start <= end;) if (tolower (input[start++]) != tolower (input[end--])) return false;

```

Comment:
instead of converting a string to lower better convert it using filter char and pass